testdata/p4_16_samples_outputs/issue2221-bmv2-frontend.p4

```diff
@@ -66,9 +55,8 @@ control ingress(inout Headers h, inout Meta m, inout standard_metadata_t sm) {
             hasReturned_2 = true;
             retval_2 = 16w2;
             h.eth_hdr.eth_type = eth_type_2;
-            tmp_6 = retval_2;
+            tmp_4 = retval_2;
```

Comment:
Why isn't this one removed too?

Reply:
I believe this is a rename of the temporary variable. Just above that line:
`@name("ingress.tmp_6") bit<16> tmp_4;`